packages/dd-trace/src/appsec/remote_config/index.js

```diff
@@ -29,6 +32,24 @@ function enable (config) {
   }
 }
 
+function enableAsmData (appsecConfig) {
+  if (appsecConfig && appsecConfig.rules === undefined) {
```

Comment:
`rc` could be null? because `disableAsmData` function checks it.

Reply:
It can be null on disable, disable can be called when enable fails. But I'll add the check here also.